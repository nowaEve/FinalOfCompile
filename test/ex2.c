void main(int n) { 
    n+=3;
    print n;
}