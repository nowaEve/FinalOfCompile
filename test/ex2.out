24 19 1 5 25 0 3 13 13 11 0 3 1 12 15 -2 13 11 22 21 1